server/proxy/proxy.go

```diff
@@ -150,6 +150,7 @@ func (p *Proxy) NotifyRemoteWebhook(app *apps.App, data []byte, webhookPath stri
 
 func (p *Proxy) GetAsset(appID apps.AppID, path string) (io.ReadCloser, int, error) {
 	app, err := p.store.App.Get(appID)
+	// app, err := p.store.Manifest.Get(appID)
```

Comment:
The PR won't work because static assets are only work for installed apps. Before I change that, I wanted to get approval for the idea.

Reply:
Fixed with [`c0c6e18` (#196)](https://github.com/mattermost/mattermost-plugin-apps/pull/196/commits/c0c6e18a95528aab5180272e6b02582cd555dbac)